### PR TITLE
Get rid of old code that retrieves "clusterID".

### PR DESCRIPTION
**Pull request by thallgren**
The clusterID was replaced with installID some time ago, the difference being that the clusterID was the UID of the "default" namespace whereas the installID is the UID of the namespace where the traffic-manager is installed.

### CODE REVIEW

File: cmd/traffic/cmd/manager/cluster/info.go [hunk 1]
```diff
@@ -377,6 +360,12 @@ func (oi *info) watchPodSubnets(ctx context.Context, namespaces []string) {
 func (oi *info) setSubnetsFromEnv(ctx context.Context) bool {
 	subnets := managerutil.GetEnv(ctx).PodCIDRs
 	if len(subnets) > 0 {
+		mgrIp, _ := netip.AddrFromSlice(oi.ManagerPodIp)
```
Comment (P0lip) [inline cmd/traffic/cmd/manager/cluster/info.go:363]: IMHO this change may deserve a separate commit.

**Review by P0lip — APPROVED** [review]

File: cmd/traffic/cmd/manager/cluster/info.go (hunk 1 above)
Comment (thallgren) [reply cmd/traffic/cmd/manager/cluster/info.go:363]: Doh, I missed this comment. But yeah, I agree. It should have been a separate commit.
